src/Microsoft.SqlTools.ServiceLayer/Admin/Contracts/GetAssociatedFilesRequest.cs

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Admin.Contracts
+{
+    /// <summary>
+    /// Params for a get database info request
+    /// </summar>
+    public class GetAssociatedFilesParams
+    {
+        /// <summary>
+        /// URI identifier for the connection to get the server folder info for
```

Comment:
Is this the whole description? or anything after for?

Reply:
Rephrased it a bit